test/__snapshots__/exportType.test.js.snap

```diff
@@ -1468,53 +1468,100 @@ a {
+var _b = \`validIdent\`;
+export { _b as \\"vIdent\\" };
+var _c = \`left\`;
+export { _c as \\"vPreDefinedIdent\\" };
+var _d = \`'content'\`;
+export { _d as \\"vString\\" };
+var _e = \`''\`;
+export { _e as \\"vString1\\" };
+var _f = \`url(https://www.exammple.com/images/my-background.png)\`;
+export { _f as \\"vUrl\\" };
+var _10 = \`url('https://www.exammple.com/images/my-background.png')\`;
+export { _10 as \\"vUrl1\\" };
+var _11 = \`url(\\"https://www.exammple.com/images/my-background.png\\")\`;
+export { _11 as \\"vUrl2\\" };
+var _12 = \`100\`;
+export { _12 as \\"vInteger\\" };
+var _13 = \`-100\`;
+export { _13 as \\"vInteger1\\" };
+var _14 = \`+100\`;
+export { _14 as \\"vInteger2\\" };
+var _15 = \`.60\`;
+export { _15 as \\"vNumber\\" };
+var _16 = \`-456.8\`;
+export { _16 as \\"vNumber1\\" };
+var _17 = \`-3.4e-2\`;
+export { _17 as \\"vNumber2\\" };
+var _18 = \`12px\`;
+export { _18 as \\"vDimension\\" };
+var _19 = \`100%\`;
+export { _19 as \\"vPercentage\\" };
+var _1a = \`#fff\`;
+export { _1a as \\"vHex\\" };
+var _1b = \` /* comment */\`;
+export { _1b as \\"vComment\\" };
+var _1c = \`rgb(0,0,0)\`;
+export { _1c as \\"vFunction\\" };
+var _1d = \`U+0025-00FF\`;
+export { _1d as \\"vUnicodeRange\\" };
+var _1e = \`ozGmfTedr1GnFJDWqNUH\`;
+export { _1e as \\"ghi\\" };
+var _1f = \`zchqshjqLbPAHaRvIBET\`;
+export { _1f as \\"myClass\\" };
+var _20 = \`WZBxXqS2GytaA3IBhhnd\`;
+export { _20 as \\"other\\" };
+var _21 = \`W51zcAMuJMsNFi1CXgWr\`;
+export { _21 as \\"otherOther\\" };
+var _22 = \`KEl5ZxzNkpjfWorrBglC\`;
+export { _22 as \\"green\\" };
+var _23 = \`ecAEWh2vww9pNEdyj9Jn\`;
+export { _23 as \\"foo\\" };
+var _24 = \`CBlowYk8qiAgWWzFeXRA \${___CSS_LOADER_ICSS_IMPORT_2____NAMED___[\\"importedSimple\\"]}\`;
+export { _24 as \\"simple\\" };
+var _25 = \`c_NHnDcX1bd_kuxgsuYi \${___CSS_LOADER_ICSS_IMPORT_3____NAMED___[\\"importedRelative\\"]}\`;
+export { _25 as \\"relative\\" };
+var _26 = \`S0Kwou8pVmsENtBP3hYm \${___CSS_LOADER_ICSS_IMPORT_4____NAMED___[\\"importedRelative\\"]}\`;
+export { _26 as \\"topRelative\\" };
+var _27 = \`rq663Pq_zV0CjpwttvK4 \${___CSS_LOADER_ICSS_IMPORT_5____NAMED___[\\"importedModule\\"]}\`;
+export { _27 as \\"myModule\\" };
+var _28 = \`fadRMHArJofp7sWEbPVR \${___CSS_LOADER_ICSS_IMPORT_6____NAMED___[\\"importedAlias\\"]}\`;
+export { _28 as \\"alias\\" };
+var _29 = \`sg1HlXqlWy6l6_Wm5iA7 \${___CSS_LOADER_ICSS_IMPORT_6____NAMED___[\\"importedAlias\\"]}\`;
+export { _29 as \\"aliasDuplicate\\" };
+var _2a = \`bnRUswvicYag6u0SPnvI\`;
+export { _2a as \\"primarySelector\\" };
+var _2b = \`kEJRwpukB2OtmkGTknbU\`;
+export { _2b as \\"blackSelector\\" };
+var _2c = \`hY2PI5vC9ABuJY1nkWnf\`;
+export { _2c as \\"header\\" };
+var _2d = \`q8mv0HutzqdsVWjl8mAz\`;
+export { _2d as \\"foobarbaz\\" };
+var _2e = \`xajoqP1d3SwrjJ4WEM8g\`;
+export { _2e as \\"url\\" };
+var _2f = \`Ix5nEHiVOsWuWxdx0twz \${___CSS_LOADER_ICSS_IMPORT_7____NAMED___[\\"scssClass\\"]}\`;
+export { _2f as \\"main\\" };
```

Comment:
Also please try to merge `export` into the one object to improve the parser perfomance

Reply:
I ran some quick benchmarks and didn't see a noticeable impact between the two approaches. A project with 100,000 CSS class names will see an improvement of just 1ms by combining the export declarations. If you think that it's worth the increased complexity in css-loader, and the knock-on effects from increased complexity in plugins for this kind of performance gain then I will make the change though.